src/position/position_utils.cairo

```diff
@@ -301,9 +302,13 @@ fn get_position_pnl_usd(
         cache.pool_token_usd = cache.pool_token_amount * cache.pool_token_price;
         cache
             .pool_pnl =
-                market_utils::get_pnl(
-                    data_store, @market, @prices.index_token_price, position.is_long, true
-                );
+                u128_to_felt252(
```

Comment:
can't we just do:
```rust
market_utils::get_pnl(
                        data_store, @market, @prices.index_token_price, position.is_long, true
                    ).into()
```

Reply:
Doing that doesn't work, I've replaced my draft thingy with the `to_signed` function from the `calc` module, for greater consistency with the rest of the code